src/validate.ts

```diff
@@ -62,22 +45,9 @@ function generateQueryReq(deviceID){
     
 }
 
-function add(num){
-    var json = [{"cool": num}];
-    json.coolness = 34.33;
-    return json
-}
-
-console.log(add(4));
-console.log(add("5"));
+const deviceInfo = (query_request["inputs"][0]["payload"]);
```

Comment:
I don't think this is necessary. You shouldn't be able to depend on this field as each function should be considered stateless, without any global variable.

Reply:
+1, if you want to add a feature where the library understands previous intent context this might be part of the optional mocked HomeGraph we discussed (b/c the context would really come from the responses rather than the previous requests) or a higher-level automator that produces QUERY/EXECUTE in sequence based on the SYNC.